src/components/ApiOptionRef.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import MDXContent from '@theme/MDXContent';
+
+<<<<<<< Updated upstream
```

Comment:
I like the direction!

Looks like some conflicts might still need to be resolved.

Reply:
Should be solved now.